### PR TITLE
Opt in for monthly_updates

### PR DESCRIPTION
**Pull request by tsiology**
Adding monthly_update template checkbox and boolean support.

### CODE REVIEW

**Comment by nickolashe** [comment]
@tsiology can we change the title of the PR to reflect the feature being developed pls?

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #Install instructions for the Mac
+#tgodwin
```
Comment (nickolashe) [inline README.md]: ?

File: openCurrents/views.py [hunk 2]
```diff
@@ -804,6 +804,12 @@ def process_email_confirmation(request, user_email):
 
         # create user account
         user_account = Account(user=user, pending=1)
+
+        if form.cleaned_data['monthly_updates']:
```
Comment (nickolashe) [inline openCurrents/views.py]: looks good

Comment (nickolashe) [inline openCurrents/views.py]: @dannypernik @tsiology have we thought about how the users are going to be able to opt out if deploy this?

File: README.md (hunk 1 above)
Comment (dannypernik) [inline README.md]: We were just testing a git push. Can be removed

File: openCurrents/views.py (hunk 2 above)
Comment (dannypernik) [inline openCurrents/views.py]: @nickolashe We would have the Unsubscribe button at the bottom of any Mailchimp email. So somehow we have to get the opt-in list to Mailchimp?

File: README.md (hunk 1 above)
Comment (nickolashe) [inline README.md]: got it

File: openCurrents/views.py (hunk 2 above)
Comment (nickolashe) [inline openCurrents/views.py]: @dannypernik is there a simple way we can set up a simple opt in/out in our system? may be in profile?

Comment (nickolashe) [inline openCurrents/views.py]: this would be preferred at this point

**Pull request merged**